server/tests/AccessTypesSpec.js

```diff
@@ -0,0 +1,212 @@
+import chai from 'chai';
+import Request from 'supertest';
+import app from '../app';
+import testData from './testData';
+import db from '../models';
+
+const request = Request.agent(app);
+const expect = chai.expect;
+const superAdmin = testData.superAdminUser;
+const departmentAdmin = testData.departmentAdminUser1;
+
+let superAdminToken, departmentAdminToken;
+describe('Access Type Tests', () => {
+  before((done) => {
+    db.sequelize.sync({ force: true })
+    .then(() => done());
```

Comment:
*Nitpick:* since the callback `done` is the only statement in the `thenable`, it can just be `then(done())`